src/Raven.Client/Documents/Operations/OngoingTasks/GetTaskInfoResult.cs

```diff
@@ -79,6 +80,7 @@ public OngoingTaskSubscription()
         public string SubscriptionName { get; set; }
         public long SubscriptionId { get; set; }
         public string ChangeVectorForNextBatchStartingPoint { get; set; }
+        public Dictionary<string, string> NextBatchStartingPointChangeVectors { get; set; }
```

Comment:
I think we need to abstract the class here as well... this is for studio, so it shouldn't too problematic. I think :)

Reply:
ChangeVectorForNextBatchStartingPointPer... ?

Reply:
done